gramps/gui/configure.py

```diff
@@ -2168,8 +2150,6 @@ def activate_change_font(self, obj=None):
                 self.find.set_sensitive(True)
                 if self.choosefont:
                     self.choosefont.set_sensitive(True)
-                if self.combo:
-                    self.combo.set_sensitive(True)
```

Comment:
Also need to remove the other `if combo:` above at about 2145.

I guess you did not use a "git cherry-pick" to grab my commit ;-).  Thanks for putting up with my comments.

Reply:
done

Reply:
done